### ISSUE TITLE
Extremely verbose output during custom-object-detection training

### ISSUE THREAD

**Issue by PushkarChatterji** [OlafenwaMoses/ImageAI#267]
Hi, 
Congrats on releasing this, it certainly helps make life easier!

I had a quick question about the extremely verbose output I am getting when I run a object detection training on custom data. 

I suspect this is coming from the tf.Print() calls you are making in /imageai/Detection/Custom/yolo.py

Is there anyway to suppress this ?

Here is a screen shot of the msgs (continuous scrolling during training)...

As a result of this continuous scrolling, I am unable to track the train and eval loss reported by keras...
![Screenshot from 2019-08-05 15-25-40](https://user-images.githubusercontent.com/51086460/62464919-6e004680-b796-11e9-8eac-df531c25d192.jpg)

**Comment by OlafenwaMoses** [comment]
My sincere apologies for this bug. I will push an update to fix it.

**Comment by PushkarChatterji** [comment]
Thank you very much! I just finished a 200 epoch run for a custom training on 1 class for yolov3 and it took me 11 hours on a GTX1070.

**Comment by OlafenwaMoses** [comment]
I am really glad to know that. Keep training more detection models. I will really love to see the detection results of your trainings. You can share them with me via my email guymodscientist@gmail.com

**Comment by OlafenwaMoses** [comment]
This issue has been fixed. You can update **ImageAI** by running command below.
`pip3 install imageai --upgrade`

**Comment by OlafenwaMoses** [comment]
Hello please see an urgent bug fix update here #278

**Comment by nineclicks** [comment]
> Thank you very much! I just finished a 200 epoch run for a custom training on 1 class for yolov3 and it took me 11 hours on a GTX1070.

Can I ask what batch size you were able to use with the 1070?
